catalog/app/containers/Bucket/requests.js

```diff
@@ -337,11 +340,11 @@ const WORKFLOWS_CONFIG_PATH = '.quilt/workflows/config.yml'
 
 export const workflowsList = async ({ s3, bucket }) => {
   try {
-    const response = await latestFile({ s3, bucket, path: WORKFLOWS_CONFIG_PATH })
+    const response = await fetchFileLatest({ s3, bucket, path: WORKFLOWS_CONFIG_PATH })
     return parseWorkflows(response.Body.toString('utf-8'))
   } catch (e) {
-    if (e instanceof ErrorFileNotFound) return emptyWorkflowsConfig
-    if (e instanceof ErrorVersionNotFound) return emptyWorkflowsConfig
+    if (e instanceof errors.FileNotFound) return emptyWorkflowsConfig
+    if (e instanceof errors.VersionNotFound) return emptyWorkflowsConfig
```

Comment:
why not `if (e instanceof errors.FileNotFound || e instanceof errors.VersionNotFound)`?